app/controllers/api/audit_logs_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::AuditLogsController < Api::ApplicationController
+  skip_after_action :verify_policy_scoped, only: :index
```

Comment:
is it safe to show all audit logs, regardless of user? Because that's what this line says to do, as far as I understand.